File: 7/main.cpp
#include <bits/stdc++.h>
using namespace std;
using ll = long long;
ll ans=1;

int is_prime(ll x){
    int j = 2;
    while(j*j<=x){
        if(x%j==0) return 0;
        j += 1;
    }
    return 1;
}

int main(){
    ll cnt=1;
    while(ans<10001){
        cnt += 2;
        ans += is_prime(cnt);
    }
    cout << cnt << endl;
}

File: 7/main.py
def is_prime(number):
    i = 2
    while i*i <= number:
        if number%i==0:
            return 0
        i+=1
    return 1

cnt = 2
ans = 0
while ans<10001:
    ans += is_prime(cnt)
    cnt += 2
print(ans)